.github/workflows/build_documentation.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       commit_sha: ${{ github.sha }}
       package: transformers
       notebook_folder: transformers_doc
-      languages: de en es fr hi it ko pt tr zh ja te
+      languages: de en es fr hi it ko pt tr zh ja te ar
```

Comment:
@AhmedAlmaghz 
Can you add Arabic in its alphabetic order?

Reply:
Ok